polynomial_commitment_scheme/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+mod traits;
+use gkr_field_config::GKRFieldConfig;
+use mpi_config::MPIConfig;
+use rand::thread_rng;
+pub use traits::{
+    ExpanderGKRChallenge, PCSForExpanderGKR, PolynomialCommitmentScheme, StructuredReferenceString,
+};
+
+#[allow(clippy::type_complexity)]
+pub fn expander_pcs_init_unsafe<
+    FieldConfig: GKRFieldConfig,
+    PCS: PCSForExpanderGKR<FieldConfig>,
+>(
+    n_input_vars: usize,
+    mpi_config: &MPIConfig,
+) -> (
```

Comment:
I don't think this is quite an API for production, as the randomness consumption is ad hoc in the method... I am also keeping track of this and will traverse back later on it.